packages/financial-templates-lib/src/helpers/acrossFeesCalculator.ts

```diff
@@ -0,0 +1,62 @@
+import Web3 from "web3";
```

Comment:
this standalone file can be re-used in the front ends for computing the realized LP fee for a given RateModel and deposit.